README.md

```diff
@@ -1264,6 +1264,7 @@ _See also: [List of streaming media systems - Wikipedia](https://en.wikipedia.or
 - [Ombi](https://ombi.io/) - A content request system for Plex/Emby, connects to SickRage, CouchPotato, Sonarr, with a growing feature set. ([Demo](https://app.ombi.io/), [Source Code](https://github.com/Ombi-app/Ombi)) `GPL-2.0` `C#`
 - [OpenZiti](https://openziti.github.io/) - Fully-featured, self-hostable, zero trust overlay network. Includes a full mesh overlay network, includes 2FA support out of the box, includes clients for all major desktop/mobile OS'es. Build zero trust network principles into your application via SDKs. ([Source Code](https://github.com/openziti/ziti)) `Apache-2.0` `Go`
 - [Orchest](https://www.orchest.io/) - A new kind of IDE for Data Science. ([Demo](https://cloud.orchest.io), [Source Code](https://github.com/orchest/orchest)) `AGPL-3.0` `Docker`
+- [OpenSpeedTest](https://github.com/openspeedtest/Speed-Test) - Pure HTML5 network performance estimation tool. ([Source Code](https://github.com/openspeedtest/Speed-Test)) `CC-BY-ND 3.0` `Docker`  
```

Comment:
>  Do not add a duplicate Source code link if it is the same as the main link.

Please remove the source code link

> This is a list of [Free](https://en.wikipedia.org/wiki/Free_software) Software
>> The freedom to improve the program, and release your improvements (and modified versions in general)

> https://creativecommons.org/licenses/by-nd/3.0/
>> NoDerivatives — If you [remix, transform, or build upon](https://creativecommons.org/licenses/by-nd/3.0/#) the material, you may not distribute the modified material.

`CC-BY-ND-3.0` is not a Free/Libre license. OpenSpeedTest must be moved to `non-free.md`, unless the license is changed. Have you considered switching to a license that allows modifications/derived works?